duckduckgo/iOS
# [DuckPlayer] 16. Localization Updates

Task/Issue URL: https://app.asana.com/0/1204099484721401/1207943158975717/f

**Description**:
Updates localization based on the latest Smarling updates

**Steps to test this PR**:
- Switch to another supported language
- Go to Settings and check DuckPlayer section is translated

This PR includes some other string updates that were not properly merged.

Example (Spanish):
<img width="536" alt="Screenshot 2024-08-07 at 1 57 16 PM" src="https://github.com/user-attachments/assets/51cc66e9-e1ab-4d0e-805d-addb86bb6c20">
